app/javascript/components/admin/AdminTable.js

```diff
@@ -57,6 +59,33 @@ class AdminTable extends React.Component {
     this.csvLink = React.createRef();
   }
 
+  handleExpandClick = rowId => {
```

Comment:
Can you add a comment to this method? most other methods in this file have comments

Reply:
Also, can we move these four method after the `componentDidMount` and `componentDidUpdate`? not sure how they ended up here